test/hack/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
+
+package hack
+
+// This package imports things required by this repository, to force `go mod` to see them as dependencies
+import (
+	_ "knative.dev/eventing/test/test_images/recordevents"
```

Comment:
Thank you so much for adding these tests!!! One thing to be aware of is the new eventshub that replaces all these ones from Eventing. You can see a conversion here, I can do this as a follow up however. Just wanted to raise awareness :)
https://github.com/knative-sandbox/eventing-rabbitmq/pull/189

Reply:
i will take a look at this and fix it.